src/providers/AuthServerProvider.ts

```diff
@@ -127,6 +127,27 @@ export class AuthServerProvider {
     sso.localStorageStoreIdentity(signer, identity)
   }
 
+  /**
+   * Get the persisted account from local storage.
+   */
+  static getAccount = () => {
+    return localStorage.getItem(STORAGE_KEY_ADDRESS)
+  }
+
+  /**
+   * Clears the identity and the rest of the persisted data created by this provider.
+   */
+  static deactivate = () => {
```

Comment:
Used by the explorer website to clear data if the user decides to change accounts.